package.json

```diff
@@ -68,6 +68,7 @@
     "e2e-test": "workspace:*",
     "eslint": "^8.6.0",
     "eslint-plugin-notice": "^0.9.10",
+    "eslint-plugin-react": "^7.31.11",
```

Comment:
could make this `^7.28.0` so that we don't get unintentional duplicates (that's the version we use elsewhere).

Reply:
Yes, will change.

Reply:
Done